src/state/sagas/app/FetchInitialData/FetchInitialData.js

```diff
@@ -29,7 +29,21 @@ export function* fetchAllInitialData(action) {
       type: APPLICATION_ACTIONS_KEY.SET_PERMISSIONS_MAPPING,
       organizationPermissions: organizationPermissions,
     });
+  } catch (error) {
+    console.log(error);
```

Comment:
console.log to remove ?

Reply:
Logging the error might be useful to get details missing from the parsed error message :slightly_smiling_face: 
But I replaced `console.log` by `console.error` :+1: